### DISCUSSION TITLE
Deadline tin

### DISCUSSION THREAD

**Discussion by VuThamPhi** [VuThamPhi/Lop-10A06-Nhom-09#12] [General]
Mọi người ơi nhóm mình còn 1 việc làm Word để hoàn thành deadline nè.

**Comment by VuThamPhi** [comment]
Bài nghị luận về câu ca dao nhóm mình đã làm hôm tết tụi mình sẽ đánh lên Word rồi trang trí. Giờ mình phân công việc đi nè

**Comment by longphan090306** [comment]
T nghĩ là Mai là thành viên mới, chưa có việc nên chắc phân công word cho Mai

**Reply by Mai-10A6** [reply]
Oke nhaaa

**Reply by Mai-10A6** [reply]
Có gì mọi ngưòi hỗ trợ tui nha

**Reply by VuThamPhi** [reply]
Vậy Mai làm phần trang trí Word rồi mọi người còn lại vô coi có gì sai sót rồi chỉnh sửa cho bài hoàn chỉnh nha.

**Reply by Mai-10A6** [reply]
À mà tui hỏi cái, phần nội dung gửi lên là copy ra word xong trang trí với yêu cầu của thầy thôi đúng ko

**Reply by longphan090306** [reply]
Thiết kế theo format thầy hướng dẫn nha

**Reply by Mai-10A6** [reply]
Okeee

**Reply by Mai-10A6** [reply]
Mà khi nào dead z

**Reply by VuThamPhi** [reply]
31/3 nha mà ráng tuần này làm rồi mình chỉnh sửa cho xong sớm rồi nộp thầy luôn

**Reply by VuThamPhi** [reply]
Phần nào làm mà không biết cứ hỏi nha Mai mọi người phụ giúp cho

**Reply by Mai-10A6** [reply]
Vậy thì chắc ngày hoặc thứ 7 t ngồi làm rồi t gửi lên cho mọi người xem thử nha

**Reply by Mai-10A6** [reply]
Xong chỉnh lại gid thì kêu t nha

**Reply by VuThamPhi** [reply]
Oke nếu được cn tuần này Mai gửi đi coi rồi chỉnh lại luôn
